doc/ref/modules/all/index.rst

```diff
@@ -373,6 +373,7 @@ execution modules
     postfix
     postgres
     poudriere
+    power_dns
```

Comment:
Any reasons to name it with underscore, I think naming it as `powerdns` would be more appropriate

Reply:
Renamed.